src/style.js

```diff
@@ -207,15 +199,13 @@ export default class StyleClient {
    * @throws Error if request fails
    */
   async assignStyleToLayer (qualifiedName, styleName, workspaceStyle, isDefaultStyle) {
```

Comment:
Please rename `workspaceStyle` to `workspace`. Knowing that this was not introduced in here, but it would be good to harmonize once we are touching this.

Reply:
done in [0b8183d](https://github.com/meggsimum/geoserver-node-client/pull/57/commits/0b8183df74ccd3143dd327906f210305f395f9ae)